language-support/scala/bindings-akka/src/test/scala/com/digitalasset/ledger/client/binding/retrying/CommandRetryFlowUT.scala

```diff
@@ -91,7 +99,7 @@ class CommandRetryFlowUT extends AsyncWordSpec with Matchers with AkkaTest {
       submitRequest(Code.OK_VALUE, Instant.ofEpochSecond(45)) map { result =>
         result.size shouldBe 1
         result.head.context.nrOfRetries shouldBe 0
-        result.head.value.status.get.code shouldBe Code.OK_VALUE
+        result.head.value shouldBe Symbol("right")
```

Comment:
Please use `matchPattern` rather than symbols. Much more type-safe.

```suggestion
        result.head.value should matchPattern { Right(_) => }
```

You could potentially use `inside(result)` in this test and subsequent ones to make it terser and get a better error message on failure (if, e.g. there are 2 results):

```scala
        inside(result) { case Seq(Ctx(context, Right(_), _) =>
          context.nrOfRetries shouldBe 0
        }
```